example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -214,31 +197,23 @@ auto main() -> int
     // elements of a buffer directly, but
     // you can get the pointer to the memory
     // (getPtrNative).
```

Comment:
Beside that the comment is outdated, I want make a suggestion for the comment. I think it would make sense, to write, that `access()` has readWriter permission and there are also a read and write accessor. The first thing, which I thought was, why not direct use `alpaka::access(hostBuffer)(z,y,x)` (I think is not good for performance) or better `hostBuffer(z,y,x)`?

Reply:
Thx for catching this! I updated the comment. That `access()` gives RW permissions is documented at the function's declaration site.
Using `alpaka::access(hostBuffer)(z,y,x)` in host code is perfectly fine if you have access to the buffer's memory there ;)

Reply:
Is there a section about the accessors in the readthedocs? Ask because I saw the doxygen documentation but I think, doxygen is not a good documentation to explain concepts. Personal, I don't read the API documentation if I want to introduce my self in a new librarie. We should do this in the readthedocs (better way) or examples (okay).

Reply:
I added a short section to the readthedocs documentation.